frontend/packages/component-library/src/cms/heroBanner/HeroBanner.tsx

```diff
@@ -0,0 +1,111 @@
+import classNames from 'classnames';
+import {ReactNode, HTMLAttributes} from 'react';
+
+import {LinkButton, LinkButtonProps} from '@/button';
+import {Theme} from '@/common/contexts';
+import Image, {ImageProps} from '@/image';
+import {Heading1, BodyOneText, BodyTwoText} from '@/typography';
+import Video, {VideoProps} from '@/video';
+
+import moduleStyles from './heroBanner.module.scss';
+
+export interface HeroBannerProps extends HTMLAttributes<HTMLElement> {
+  /** HeroBanner heading */
+  heading: string | ReactNode;
+  /** HeroBanner subHeading */
+  subHeading?: string | ReactNode;
+  /** HeroBanner description */
+  description?: string | ReactNode;
+  /** HeroBanner image */
+  imageProps?: ImageProps;
+  /** HeroBanner video */
+  videoProps?: VideoProps;
+  /** HeroBanner link */
+  buttonProps?: LinkButtonProps;
+  /** HeroBanner custom background url */
+  backgroundImageUrl?: string;
+  /** HeroBanner theme value.
+   *  If you're using backgroundImageUrl - you should make sure you set correct theme value to HeroBanner.
+   *  */
+  'data-theme'?: Theme;
+  /** Whether to show the background color */
+  removeBackground?: boolean;
+  /** HeroBanner whether show with wide text (text is wider than image) */
+  withWideText?: boolean;
+  /** HeroBanner partner prop */
+  partner?: {title: string; logo: ImageProps};
+  /** HeroBanner custom className  */
+  className?: string;
+}
+
+/**
+ * ## Production-ready Checklist:
+ *  * (✔) implementation of component approved by design team;
+ *  * (✔) has storybook, covered with stories and documentation;
+ *  * (✔) has tests: test every prop, every state and every interaction that's js related;
+ *  * (see ./__tests__/HeroBanner.test.tsx)
+ *  * (✔) passes accessibility checks;
+ *
+ * ### Status: ```Ready for dev```
+ *
+ * Design System: HeroBanner Component.
+ * Renders a Hero Banner/Section which serves as an opening section of a page.
```

Comment:
Can we make this a bit more explicit about only being used once on a page?
```suggestion
 * Renders a Hero Banner/Section which serves as an opening section of a page. There should only be one Hero Banner per page at the very top of the page under the header navigation. 
```